js/repl-worker.js

```diff
@@ -25,6 +25,7 @@ onmessage = function(e) {
       filename: 'repl',
       timeout: 1000,
       serialize: true,
+      heapGraphFilePath: "/tmp/foo.txt", /* this path will not be used, it is needed to trigger the graph computation */
```

Comment:
Hmm, it's my fault to design the option badly. The options passing to prepackSources() API should be a boolean flag instead of a path. Only prepack-cli itself should see the path. Feel free to change it in future PR.